### ISSUE TITLE
Update UtilityScripts.ini request

### ISSUE THREAD

**Issue by EmiMods** [XboxUnity/AuroraScripts#38]
Not sure if this is a proper "issue", however I was curious if the sync for what AuroraRepo hooks to so users can download scripts from their console (UtilityScripts.ini) might be broken? Or perhaps if there is a different vetting/system for determining what gets referenced there?

Not sure who owns it, but figured I'd ask since I noticed the two scripts I recently released weren't available through Aurora Repo Browser :)

**Comment by Swizzy** [comment]
@Ste1io has been working on fixing that, unfortunately alot of other things have gotten in the way lately.

Hopefully he can complete the work soonish so it gets done automatically, until then he has had to do things manually. there's been quite a few recently added scripts actually.

**Comment by EmiMods** [comment]
Understandable, thank you for the info!

**Comment by Ste1io** [comment]
Which scripts are you referencing specifically, @EmiMods ?

**Comment by Swizzy** [comment]
These ones;

https://github.com/XboxUnity/AuroraScripts/tree/master/UtilityScripts/ExportGamesList

https://github.com/XboxUnity/AuroraScripts/tree/master/UtilityScripts/FixClonedDrive

But also this one;

https://github.com/XboxUnity/AuroraScripts/tree/master/UtilityScripts%2FDiscRipper

**Comment by EmiMods** [comment]
Yes that's correct, sorry I didn't see those last to messages until now.

**Comment by Ste1io** [comment]
Ok,  seeI the pull requests now. Since the PRS have been closed, I will need to push these manually. Will get to that tonight or early this weekend. My PR action to automate the process should be ready to go around the same time, so this shouldn't be an issue in the future. Thanks for your patience @EmiMods. I'm reopening this issue and will close it as soon as the update is live.
